app/jobs/setup_metadata_from_mets_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class SetupMetadataFromMetsJob < ApplicationJob
+  queue_as :metadata
+
+  def perform(parent_object, current_batch_process, current_batch_connection = parent_object.current_batch_connection)
+    parent_object.current_batch_process = current_batch_process
+    parent_object.current_batch_connection = current_batch_connection
+    parent_object.generate_manifest = true
+    parent_object.default_fetch(current_batch_process, current_batch_connection)
+    parent_object.create_child_records_from_mets
```

Comment:
if parent object gets permanent awareness of its "from_mets-ness" then I think `create_child_records` can just call `create_child_records_from_mets` or `create_child_records_from_ladybird` intelligently inside the parent object and this job no longer needs to happen separately =-)